AttitudeManager/FlightModes/Src/manual.cpp

```diff
@@ -3,10 +3,28 @@
 
 namespace AM {
 
-AttitudeManagerInput Manual::run(const AttitudeManagerInput& input) {
+AttitudeManagerInput Manual::run(const AttitudeManagerInput& input, AxisPIDs _pids) {
     return input;
 }
 
 void Manual::updatePid() {}
+void Manual::updatePidGains(PIDController _axis, GainTerm whichGain, float desiredGain) {
```

Comment:
You shouldn't need to pass in a PIDController class - you should be able to pass in the "axis" that you would like to tune ('whichAxis', for example), and your function should be able to grab the flightmodes private PID class.

This follows the "no external access to the PID's". Each class owns it's own PID controllers.

Also - "whichGain" or "whichAxis" probably aren't my top picks for variable names, I'd rather `pid_axis` and `pid_gain_term`.

I'm not sure what our code style says for snake vs camel case, but whatever it says goes good w/me.